app/_locales/es_419/messages.json

```diff
@@ -528,184 +789,307 @@
-    "message": "RPC de URL no válido"
+    "message": "Dirección URL de RPC no válida"
   },
   "invalidSeedPhrase": {
-    "message": "Frase de inicialización no válida"
+    "message": "Frase de recuperación no válida"
+  },
+  "ipfsGateway": {
+    "message": "Puerta de enlace de IPFS"
+  },
+  "ipfsGatewayDescription": {
+    "message": "Escriba la dirección URL de la puerta de enlace de IPFS CID para usar la resolución de contenido de ENS."
   },
   "jsonFile": {
-    "message": "Archivo JSON",
-    "description": "format for importing an account"
+    "message": "Archivo JSON"
   },
   "knownAddressRecipient": {
     "message": "Dirección de contrato conocida."
   },
+  "knownTokenWarning": {
+    "message": "Esta acción editará tokens que ya estén enumerados en la cartera y que se pueden usar para engañarlo. Apruebe solo si está seguro de que quiere cambiar lo que representan estos tokens."
+  },
   "kovan": {
     "message": "Red de prueba Kovan"
   },
+  "lastConnected": {
+    "message": "Última conexión"
+  },
   "learnMore": {
     "message": "Más información"
   },
   "ledgerAccountRestriction": {
-    "message": "Necesitas utilizar tu cuenta anterior para agregar una nueva."
+    "message": "Debe usar su última cuenta antes de poder agregar una nueva."
   },
   "letsGoSetUp": {
-    "message": "¡Eso, pongámonos en marcha!"
+    "message": "Sí, vamos a establecer la configuración."
   },
   "likeToAddTokens": {
-    "message": "¿Te gustaría agregar estos tokens?"
+    "message": "¿Le gustaría agregar estos tokens?"
   },
   "links": {
-    "message": "Enlaces"
+    "message": "Vínculos"
   },
   "loadMore": {
     "message": "Cargar más"
   },
   "loading": {
-    "message": "Cargando..."
+    "message": "Cargando…"
   },
   "loadingTokens": {
     "message": "Cargando tokens…"
   },
+  "localhost": {
+    "message": "Host local 8545"
+  },
   "lock": {
-    "message": "Cerrar sesión"
+    "message": "Bloquear"
+  },
+  "lockTimeTooGreat": {
+    "message": "El tiempo de bloqueo es demasiado largo"
   },
   "mainnet": {
     "message": "Red principal de Ethereum"
   },
   "max": {
     "message": "Máx."
   },
+  "memo": {
+    "message": "memorándum"
+  },
   "memorizePhrase": {
-    "message": "Memoriza esta frase."
+    "message": "Memorice esta frase."
   },
   "message": {
     "message": "Mensaje"
   },
+  "metaMaskConnectStatusParagraphOne": {
+    "message": "Ahora tiene más control sobre las conexiones de su cuenta en MetaMask."
+  },
+  "metaMaskConnectStatusParagraphThree": {
+    "message": "Haga clic en él para administrar las cuentas conectadas."
+  },
+  "metaMaskConnectStatusParagraphTwo": {
+    "message": "El botón de estado de la conexión muestra si el sitio web que visita está conectado a la cuenta seleccionada actualmente."
+  },
   "metamaskDescription": {
-    "message": "Te estamos conectando a Ethereum y a la web descentralizada."
+    "message": "Conectándolo a Ethereum y a la Web descentralizada."
```

Comment:
I understand dialectical loismo dialectical concerns, but leismo allows gender-neutral expressions. Correcting throughout to avoid gendering the user as male.